src/gui/application.cpp

```diff
@@ -104,16 +89,9 @@ namespace {
 Application::Application(int &argc, char **argv)
     : SharedTools::QtSingleApplication(Theme::instance()->appName(), argc, argv)
     , _gui(nullptr)
+    , _parser()
```

Comment:
This is unnecessary.

Reply:
Hey, this is true. I am just used to write down all members in the order of initialization. If it is your guideline to leave default constructed objects I will remove them. May not change anything in this case.